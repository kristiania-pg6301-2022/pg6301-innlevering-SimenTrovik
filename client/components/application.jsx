import { BrowserRouter, Route, Routes } from "react-router-dom";
import { FrontPage } from "./frontPage.jsx";
import { QuizPage } from "./quizPage.jsx";
import React from "react";

export function Application() {
  return (
    <BrowserRouter>
      <Routes>
        <Route path={"/"} element={<FrontPage />} />
        <Route path={"/quizPage"} element={<QuizPage />} />
      </Routes>
    </BrowserRouter>
  );
}
