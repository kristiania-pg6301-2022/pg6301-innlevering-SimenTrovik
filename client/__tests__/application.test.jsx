import React from "react";
import ReactDOM from "react-dom";
import { Application } from "../components/application.jsx";

describe("application", () => {
  it("renders as expected", () => {
    const element = document.createElement("div");
    ReactDOM.render(<Application />, element);
    expect(element).toMatchSnapshot();
  });
});
