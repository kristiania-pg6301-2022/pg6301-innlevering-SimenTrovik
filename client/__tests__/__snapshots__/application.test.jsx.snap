// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`application renders as expected 1`] = `
<div>
  <div>
    <h1>
      Welcome to Quiz'ter
    </h1>
    <div>
      Click to start a quiz
    </div>
    <a
      href="/quizPage"
    >
      Quiz
    </a>
  </div>
</div>
`;
